src/main/java/binaries/limbert/BinaryArray.java

```diff
@@ -18,6 +18,12 @@ public BinaryArray(int[] binaries) {
      * Returns the decimal numeric value equivalent to the list of binaries stored
      */
     public int ConvertToDecimalInt() {
-        throw new UnsupportedOperationException("Not supported yet.");
+        //throw new UnsupportedOperationException("Not supported yet.");
+    	int longitud = binaries.length;
+		int dec = 0;
```

Comment:
be aware of the abbreviated names, sometimes they are not as meaningful as their non-abbreviated version